frontend/src/views/SqlEditor/TabListContainer.vue

```diff
@@ -164,10 +187,10 @@ const labelState = reactive({
 });
 const labelInputRef = ref<HTMLInputElement>();
 
-const tabListOptions = computed(() => {
+const localTabList = computed(() => {
   return tabList.value.map((tab: TabInfo) => {
     return {
-      label: tab.label,
+      label: tab.name,
```

Comment:
also change to name here?

Reply:
can not change to name here. I use the `PopSelect` components in `navie-ui`, which options structure is

```json
{
  label: 'xxx',
  value: 'xxx'
}
```